packages/babel-plugin-react-jsx/src/TransformJSXToReactBabelPlugin.js

```diff
@@ -24,6 +24,42 @@
 'use strict';
 
 const esutils = require('esutils');
+const helperModuleImports = require('@babel/helper-module-imports');
+const isModule = helperModuleImports.isModule;
+const addNamespace = helperModuleImports.addNamespace;
+const addNamed = helperModuleImports.addNamed;
+const addDefault = helperModuleImports.addDefault;
```

Comment:
```suggestion
const {isModule, addNamespace, addNamed, addDefault} = require('@babel/helper-module-imports');
```
I love destructuring imports :)

Reply:
Oops I meant to change this but I forgot. Good catch!